src/actions/index.js

```diff
@@ -364,8 +374,7 @@ export function notifyObservers(payload) {
 }
 
 function updateOutput(
-    outputComponentId,
-    outputProp,
+    outputIdAndProp,
```

Comment:
Modified the dash side handling of dispatch to only take the string key of the output.